owslib/coverage/wcs100.py

```diff
@@ -38,17 +38,18 @@ def __getitem__(self, name):
         else:
             raise KeyError("No content named %s" % name)
 
-    def __init__(self, url, xml, cookies, auth=None):
+    def __init__(self, url, xml, cookies, auth=None, timeout=30):
         super(WebCoverageService_1_0_0, self).__init__(auth)
         self.version = '1.0.0'
         self.url = url
         self.cookies = cookies
+        self.timeout = timeout
```

Comment:
@RemiDesgrange I think there is a bug here.  You are setting `self.timeout = timeout` for `wcs100.py` only but not for all the other files.

This might explain why we are getting `AttributeError: 'WebCoverageService_2_0_1' object has no attribute 'timeout'` in https://github.com/Ouranosinc/raven/pull/348#issuecomment-770118401.